{bio}[GCC/12.3.0] PLINK v2.00a3.7

benczaja opened the pull request:
(created using `eb --new-pr`)

akesandgren commented:
@boegelbot Please test @ jsc-zen2

akesandgren commented on easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.7-GCC-12.3.0.eb:
```diff
@@ -0,0 +1,39 @@
+checksums = ['145717350205f5562a01292a0fcbd617b7500758f20cad0393c7cc54665a614e']
+
+dependencies = [
+    ('FlexiBLAS', '3.3.1'),
```
Drop FlexiBLAS and use gfbf/2023a as above instead

akesandgren commented on easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.7-GCC-12.3.0.eb:
```diff
@@ -0,0 +1,39 @@
+
+# ignore suggested Makefile in build_dynamic, the dev Makefile is better
+buildopts = 'BLASFLAGS="-lflexiblas" BLASFLAGS64="-lflexiblas"'
+buildopts += ' LDFLAGS="$LIBS -ldeflate -lz -lzstd" ZLIB="" CSRC="include/SFMT.c"'
```
Since you have -lz here zlib should also be explicitly mentioned in the dependencies.

akesandgren reviewed (changes requested):
In general, please make this more like the PLINK-2.00a3.7-foss-2022a.eb, but based on gfbf/2023a instead.
It's missing PLINK1 at the moment.

benczaja commented:
Hi @akesandgren ,

Sorry I am still cathcin up to the new naming schemes. Since I renamed the whole easyconfig (which now is gfbf), I just opened up a new PR so it can be easily found back again https://github.com/easybuilders/easybuild-easyconfigs/pull/20224

I suggest that you close this PR and review https://github.com/easybuilders/easybuild-easyconfigs/pull/20224

I have incorporated all your suggested edits

akesandgren commented:
Closing, better one available in #20224
